src/main/java/com/gudimov/movieland/controller/MovieController.java

```diff
@@ -39,5 +40,20 @@
             return new ResponseEntity<>(movieJson, HttpStatus.OK);
         }
     }
+
+    @RequestMapping(value = "/random", method = RequestMethod.GET, produces = "application/json;charset=UTF-8")

```

Comment:
produces and method can be applied in @RequestMapping in class definition

Reply:
Fixed